### PR TITLE
[WIP] Switch to manageiq-password instead of gems-pending

### PR DESCRIPTION
**Pull request by agrare**
Use the extracted manageiq-password gem instead of gems-pending

### CODE REVIEW

**Comment by martinpovolny** [comment]
@Fryguy, @chessbyte : seems this one is on the critical path to fix master and you 2 are the only ones with the permissions.

This cannot be fixed from this side of the ocean.

Or can @NickLaMuro also merge this one?

**Comment by agrare** [comment]
@martinpovolny I think its better to rollback the change to smartstate before applying any more of these

File: manageiq-schema.gemspec [hunk 1]
```diff
@@ -16,12 +16,13 @@ Gem::Specification.new do |s|
   s.files = Dir["{db,lib}/**/*", "LICENSE.txt", "Rakefile", "README.md"]
 
   s.add_dependency "activerecord-id_regions", "~> 0.2.2"
+  s.add_dependency "manageiq-password", "~> 0.1.0"
   s.add_dependency "more_core_extensions", "~> 3.5"
   s.add_dependency "pg", "~> 0.18.2"
   s.add_dependency "pg-pglogical", "~> 2.1.1"
   s.add_dependency "rails", "~> 5.0.2"
 
-  s.add_dependency "manageiq-gems-pending"  # This is just for MiqPassword for now
```
Comment (bdunne) [inline manageiq-schema.gemspec]: This should also be removed from the `Gemfile`

**Comment by bdunne** [comment]
I can merge this when ready

**Pull request closed without merging**